messaging/WFLY-5286_add_runtime_journal_type_attribute.adoc

```diff
@@ -0,0 +1,71 @@
+= <INSERT TITLE HERE>
+:author:            Emmanuel hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+When configuring the use of AIO for journal persistence, the Artemis server may fall back to NIO if the libaio binaries are not located. This state is not displayed to the user so he doesn't know the effective journal type that is used.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-5286
+
+=== Related Issues
+
```

Comment:
Please add https://issues.jboss.org/browse/EAP7-850 here.

Reply:
I'm not sure this is required as this is community documentation

Reply:
We always mentioned related EAP7 issues here. For example:
https://github.com/wildfly/wildfly-proposals/blob/master/messaging/WFLY-10326_messaging_global_resource_usage.adoc
https://github.com/wildfly/wildfly-proposals/blob/master/messaging/WFLY-11829_artemis_clusters_with_http_load_balancer.adoc
We always mentioned related EAP7 issues here.